tests/Unit/DataStructures/DataBox/Test_DataBox.cpp

```diff
@@ -2674,6 +2674,33 @@ struct FromTaggedTuple : Tag, db::ReferenceTag {
 };
 /// [databox_reference_tag_example]
 }  // namespace test_databox_tags
+
+void test_reference_item() noexcept {
```

Comment:
This is a useful test, but also seems a little confusing in a commit about the `FromGlobalCache` tag.